### PR TITLE
Added domain

### PR DESCRIPTION
**Pull request by js-kyle**

### CODE REVIEW

File: domains/kylemartin [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Kyle Martin",
+  "repo": "https://github.com/js-kyle/js-kyle.github.io",
+  "owner": {
+    "username": "js-kyle"
```
Comment (MaskDuck) [inline domains/kylemartin:5]: i would like to see an email here. if you dont have, or dont want to leak, just leave it `{"email": ""}`.
`

File: domains/kylemartin [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  },
+  "record": {
+    "CNAME": "ks-kyle.github.io"
```
Comment (MaskDuck) [inline domains/kylemartin:8]: i wonder if you have made a typo here.

**Review by MaskDuck — CHANGES_REQUESTED** [review]
also i would like the file to have the extension .json, not the one without extension. please read the [docs](https://github.com/is-a-dev/register/blob/main/docs/domains-json.md)!

**Comment by js-kyle** [comment]
@MaskDuck thanks, apologies, that was a bit of a mess. fixed

**Review by MaskDuck — APPROVED** [review]
gud

**Comment by MaskDuck** [comment]
@js-kyle np

**Review by hackermondev — APPROVED** [review]

**Pull request merged**